test/testdata/googleapis_test/gapic/lang/common.yaml

```diff
@@ -1,5 +1,6 @@
 common:
   toolkit_path: test/fake-repos/toolkit
+  staging_repo_dir: '${REPOROOT}/test/testdata/test_output/final'
```

Comment:
oh! Maybe put 'staging' in the directory name so that it's easier for later maintainers to see the link.